app/code/Magento/Newsletter/Test/Mftf/Data/NewSubscriberData.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please remove this and use existing dataset
app/code/Magento/Customer/Test/Mftf/Data/CustomerData.xml